src/main/scala/viper/gobra/frontend/Parser.scala

```diff
@@ -221,7 +222,7 @@ object Parser {
       val r = s"($finalTokenRequiringSemicolon)((?:$ignoreComments|$ignoreWhitespace)*)$$".r
       // group(1) contains the finalTokenRequiringSemicolon after which a semicolon should be inserted
       // group(2) contains the line's remainder after finalTokenRequiringSemicolon
-      r.replaceAllIn(line, m => m.group(1) ++ ";" ++ m.group(2))
+      r.replaceAllIn(line, m => StringEscapeUtils.escapeJava(m.group(1) ++ ";" ++ m.group(2)))
```

Comment:
what is this doing?

Reply:
According to the [doc](https://commons.apache.org/proper/commons-text/javadocs/api-release/org/apache/commons/text/StringEscapeUtils.html#escapeJava(java.lang.String)), this escapes the special chars in a string according to the java string rules e.g.
```
 input string: He didn't say, "Stop!"
 output string: He didn't say, \"Stop!\"
```

Without this modification, the following line of Gobra code would cause an error during the preprocess step
```go
s3 := "\""
```
Error:
```
[info] Gobra has found 1 error(s):
[info] 	<13:10> strings0.gobra:13:10:error: '}' expected but '"' found
[info] 	s3 := """;
```